plugins/memcached/CMakeLists.txt

```diff
@@ -31,20 +31,35 @@ if (MEMCACHED)
         # until then, we build the required libraries from source
 
         set(LIBMEMCACHED_VERSION "1.0.10")
+        set(LIBMEMCACHED_URL https://launchpad.net/libmemcached/1.0/${LIBMEMCACHED_VERSION}/+download/libmemcached-${LIBMEMCACHED_VERSION}.tar.gz)
+        file(DOWNLOAD
```

Comment:
In addition, shoudn't the minversion be passed in to ADD_PLUGIN and thus have the entirety of the download in FindLIBMEMCACHED.cmake?